integration-libs/cdc/root/service/cdc-js.service.ts

```diff
@@ -574,11 +574,34 @@ export class CdcJsService implements OnDestroy {
           if (currentEmail !== userDetails.uid) {
             this.logoutUser();
           }
+          this.handleProfileUpdateTostMessage(response);
         });
       });
     }
   }
 
```

Comment:
HIGH: there is an if block from line 566-582, for this if block.add an 'else' block to it like below:
`else{this.handleProfileUpdateTostMessage();}` so that, error message can be displayed in case you don't receive any response from CDC.

Reply:
Done